docs/usage/create.rst

```diff
@@ -74,3 +74,14 @@ Examples
     $ cd /home/user/Documents
     # The root directory of the archive will be "projectA"
     $ borg create /path/to/repo::daily-projectA-{now:%Y-%m-%d} projectA
+
+    # Use external command to determine files to archive
+    # Use --paths-from-stdin with find to only backup files less than 1MB in size
+    $ find ~ -size -1000k | borg create --paths-from-stdin /path/to/repo::small-files-only
+    # Use --paths-from-command with find to only backup files from a given user
+    $ borg create --paths-from-command -- /path/to/repo::joes-files find ~ -user joe
+    # Use --paths-from-stdin with --paths-delimiter to backup files listed in a csv file
+    $ cat ~files-to-backup.csv | borg create \
```

Comment:
it needs to have a `/` after the `*`.

also, there is still the "one row" issue. when thinking of csv, on usually has items distributed over rows, not over columns (which are rather for other attributes of same item).

Reply:
Read up on the one find option and now I see why someone would want to do that.